### ISSUE TITLE
Constrained/unconstrained quotes

### ISSUE THREAD

**Issue by kikijiki** [asciidoctor/kramdown-asciidoc#6]
Quotes should be doubled when there is no space before/after the text.

In my case the documents are in Japanese, so there is no space between words. An example:

Markdown text: ```計算の過程で`NaN`や`Inifinite`が出た際に```
Kramdoc output: ```計算の過程で`NaN`や`Inifinite`が出た際に```
Expected output: ```計算の過程で``NaN``や``Inifinite``が出た際に```

**Comment by mojavelinux** [comment]
Indeed, this is one of the remaining conversions that need to be done. However, before I do that, I'm refactoring the code to make it more flexible. Once that is done, adding support for unconstrained formatting should be possible.

**Comment by mojavelinux** [comment]
I've added basic support. There may still be edge case. If you find them, please submit a separate issue that is specific to the case.

**Comment by kikijiki** [comment]
Cool, thank you for fixing this.
